common-lib/src/cmr/common/api/web_server.clj

```diff
@@ -19,6 +19,19 @@
   ring jetty adapter default of 8."
   8)
 
+(declare use-web-compression?)
+(defconfig use-web-compression?
+  "Indicates whether the servers will use gzip compression. Disable this to
+  make tcpmon usable"
+  {:default true
+   :type Boolean})
+
+(declare use-access-log)
+(defconfig use-access-log
```

Comment:
NOTE: I'm turning this to true which was the default in web_server.clj. This means that dev-system will log now unless you set an env, which I don't think is a big deal.